src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    default Object getArea() {
+        return null;
+    }

```

Comment:
1. never return null
2. never use Object when it's possible to declare exact type
3. better avoid default methods, here there's absolutely no need to use them
4. bad naming - if interface is Calculator, why method name sounds like `get...`
```suggestion
    double calculateArea();
```